lib/ama_layout/version.rb

```diff
@@ -1,3 +1,3 @@
 module AmaLayout
-  VERSION = '3.2.0'
+  VERSION = '4.0.0.pre1'
```

Comment:
Yea - it really helps bundler tell when a gem has changed. You could even do `4.0.0.kayt` if you wanted.